docker-compose.yml

```diff
@@ -1,19 +1,29 @@
 services:
+  minio:
+    image: minio/minio:RELEASE.2023-08-09T23-30-22Z
+    env_file: .aws.env
+    ports:
+      - "9001:9001"
+      - "9442:9000"
+    command: server /data --console-address ":9001"
+    volumes:
+      - s3minio:/data
+      - ./docker-minio-storage:/scripts
```

Comment:
just wondering if we could run this scripts as part of the container "command:"? We already check if it exists so repeat runs shouldnt do any harm and it removes this manual step

Reply:
I've spend last 2 days to get the include the script of the container(like entrypoint, Dockerfile...). But it doesn't work as minio needs to run first to create any buckets.